### PR TITLE
fix(modal): new slots

### PR DESCRIPTION
**Pull request by theJohnnyMe**
Updating modal slots

### CODE REVIEW

File: src/components/Modal.tsx [hunk 1]
```diff
@@ -25,22 +25,18 @@ const Modal = ({ selector, size, prevent }: ModalProps) => {
       size={size}
       actions="static"
     >
-      <h5 className="tds-modal-headline" slot="tds-modal-headline">
+      <h5 className="tds-modal-headline" slot="headler">
```
Comment (nathalielindqvist) [inline src/components/Modal.tsx:28]:
```suggestion
      <h5 className="tds-modal-headline" slot="header">
```

Comment (alexandereneroth) [reply src/components/Modal.tsx:28]: Ah, NVM just realised this is the implementation, that ship has sailed for now ⛵️

Comment (alexandereneroth) [reply src/components/Modal.tsx:28]: Also it makes more sense with heading when its the implementation

Comment (nathalielindqvist) [reply src/components/Modal.tsx:28]: Apparently we aren't fully aligned on the header vs heading and subheader vs subheading naming for our slots, we have both but in different components.

Comment (theJohnnyMe) [reply src/components/Modal.tsx]: Where did you notice things are off?

**Review by nathalielindqvist — APPROVED** [review]
LGTM

**Pull request merged**